src/test/java/edu/wpi/first/outlineviewer/view/NetworkTableTreeTest.java

```diff
@@ -117,7 +119,7 @@ void testDeleteNestedEntryWithSiblings() {
    */
   private void waitForNtcoreEvents() {
     NetworkTablesJNI
-        .waitForEntryListenerQueue(NetworkTableUtilities.getNetworkTableInstance().getHandle(), -1);
+        .waitForEntryListenerQueue(NetworkTableUtilities.getNetworkTableInstance().getHandle(), 3);
```

Comment:
I'm okay with a timeout here, but why not use NetworkTableInstance.waitForEntryListenerQueue() instead of the JNI function?